assets/extensions/use-sensei-pro-extension.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { EXTENSIONS_STORE } from './store';
+
+export default function useSenseiProExtension() {
```

Comment:
I created this hook based on [this component](https://github.com/Automattic/sensei/blob/a21c5162e31bc88079a76301fe6740aae2b6699d/assets/extensions/featured-product-sensei-pro.js) that I found inside the Sensei Core repository. I accept any feedback about how to improve this, as I need to get the extension's detail both on the Wizard component (to remove the Paid Course upsell if Sensei Pro extension is installed) as well as on the Course Upgrade Step (to get the price dynamically).

Reply:
I was thinking that maybe another selector in the store to get the Sensei Pro extension would fit better for this purpose. But I think it also works well.

Reply:
Changed in [9635974](https://github.com/Automattic/sensei/pull/5169/commits/9635974f54dd32a84ab5a5251adbaa62be67d7d9). Thanks!